pact/coin-contract/coin.pact

```diff
@@ -451,6 +461,105 @@
           (credit create-account create-account-guard quantity))
         ))
     )
+
+
+  ; --------------------------------------------------------------------------
+  ; Coin allocations
+
+  (defschema allocation-schema
+    @doc "The coin allocation schema for genesis lockups"
+    @model [ (invariant (>= balance 0.0)) ]
+
+    balance:decimal
+    date:time
+    guard:guard
+    redeemed:bool)
+
+  (deftable allocation-table:{allocation-schema})
+
+  (defun create-allocation-account (account:string date:time guard-ref:string amount:decimal)
+    @doc "Add an entry to the coin allocation table"
```

Comment:
Might want to start doc'ing required caps, like "Requires ALLOCATION".

Reply:
Removing allocation caps to simplify things